c7n/resources/rdscluster.py

```diff
@@ -599,3 +600,58 @@ def process_snapshot_set(self, client, snapshots_set):
             except (client.exceptions.DBSnapshotNotFoundFault,
                     client.exceptions.InvalidDBSnapshotStateFault):
                 continue
+
+
+@RDSCluster.filter_registry.register('consecutive-snapshots')
+class ConsecutiveSnapshots(Filter):
+    """Returns RDS clusters where number of consective daily snapshots is equal to/or greater
+     than n days.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: rdscluster-daily-snapshot-count
+                resource: rds-cluster
+                filters:
+                  - type: consecutive-snapshots
+                    days: 7
+                    exclude_engine: ["postgres"]
+    """
+    schema = type_schema('consecutive-snapshots', days={'type': 'number', 'minimum': 1},
+     exclude_engine={'type': 'array', 'items': {'type': 'string'}}, required=['days'])
+    permissions = ('rds:DescribeDBClusterSnapshots', 'rds:DescribeDBClusters')
+
+    def enhance_resources(self, resources):
+        exclude_engine = self.data.get('exclude_engine', [])
+        client = local_session(self.manager.session_factory).client('rds')
+        results = []
+
+        for resource in resources:
+            if resource['Engine'] not in exclude_engine:
```

Comment:
We would typically filter by engine using a separate `value` filter rather than baking that in here.

Reply:
makes sense .. fixed.